datasette/utils/__init__.py

```diff
@@ -70,6 +70,30 @@ async def await_me_maybe(value):
     return value
 
 
+def convert_list_to_sql(list, descending=False):
+    """Takes a Python list, escapes its values and returns it as comma separated values"""
+    value_list = ""
+    for item in list:
+        value_list += f"{escape_sqlite(item)}{' desc' if descending else ''}, "
+    return value_list[:-2]
+
+
+def convert_list_to_sql_is_null(list):
```

Comment:
Consider simplifying the name a bit, how about `check_nulls` and `check_not_nulls` for the other one.